packages/auth/src/core/strategies/email_and_password.ts

```diff
@@ -308,6 +308,15 @@ export async function createUserWithEmailAndPassword(
         );
         return signUp(authInternal, requestWithRecaptcha);
       } else {
+        // Fetch the password policy if the password did not meet policy requirements and there is an existing policy cached.
```

Comment:
Can you also add a comment here stating how would a customer be able to use this feature for the first time?

Reply:
Done.